src/gracenote.js

```diff
@@ -10,10 +11,10 @@ export class GraceNote extends StaveNote {
   static get SCALE() { return 0.66; }
 
   constructor(note_struct) {
-    super(Object.assign(note_struct, {
+    super(Vex.Merge({
```

Comment:
I'd like to deprecate `Vex.Merge` (and `Object.assign`) for the simpler ES6 `...` syntax. Please use the following:

```javascript
super({
  glyph_font_scale: Flow.DEFAULT_NOTATION_FONT_SCALE * GraceNote.SCALE,
  stroke_px: GraceNote.LEDGER_LINE_OFFSET,
  ...note_struct
});
```

Reply:
Yeah much cleaner ;)